backend/tests/unit/git/test_git_repository.py

```diff
@@ -124,10 +123,10 @@ async def test_init_existing_repository(git_repo_01: InfrahubRepository):
 
     # Check if all the directories are present
     assert repo.has_origin is True
-    assert os.path.isdir(repo.directory_root)
-    assert os.path.isdir(repo.directory_branches)
-    assert os.path.isdir(repo.directory_commits)
-    assert os.path.isdir(repo.directory_temp)
+    assert Path.is_dir(repo.directory_root)
+    assert Path.is_dir(repo.directory_branches)
+    assert Path.is_dir(repo.directory_commits)
+    assert Path.is_dir(repo.directory_temp)
```

Comment:
`repo.directory_*` are `Path` objects, so `repo.directory_root.is_dir()` should be fine